content/kubeone/main/tutorials/creating-clusters/_index.en.md

```diff
@@ -885,6 +900,17 @@ cloudProvider:
 
     [Network]
     public-network = "NAT Network"
+  csiconfig: |
+    [Global]
+    cluster-id = "<CLUSTER-ID>"
+    cluster-distribution = "<CLUSTER-DISTRIBUTION>"
```

Comment:
Can we document how to set those two values? The first value should be the name of the cluster, but I'm not sure about the second value.

Reply:
I basically picked this up from our kubeone init docs :D But let me take a look at this stuff.